src/code-generator/test/generator.non-interactive-cli.test.ts

```diff
@@ -7,13 +7,87 @@ let emptyFolderForATest: string;
 
 beforeEach(async () => {
   emptyFolderForATest = await testHelpers.createUniqueFolder();
+  console.log(emptyFolderForATest);
```

Comment:
```suggestion
```

Reply:
Not trivial, but in these kind of code generation tests - frequently you want to dig into the generated code so you need the path